cpp/src/slice2java/JavaUtil.cpp

```diff
@@ -780,7 +780,7 @@ Slice::JavaGenerator::output() const
 // otherwise, return the name unchanged.
 //
 string
-Slice::JavaGenerator::fixKwd(const string& name) const
```

Comment:
This function was switched from a `const instance` function to a `static` function.
There was no reason for it to not be static.